test/dogma/config_test.exs

```diff
@@ -67,7 +68,7 @@ defmodule Dogma.ConfigTest do
         override: %{ ModuleDoc => [enabled: false] }
       ) do
 
-        stdout = ExUnit.CaptureIO.capture_io(fn->
+        stdout = ExUnit.CaptureIO.capture_io(fn ->
```

Comment:
Is this style used commonly in Elixir? I've only ever seen things like `fn -> ...` and `fn(x) -> ...`. Even in this repo,  These forms seem to appear more often than `fn->`.